edit-post/store/effects.js

```diff
@@ -69,7 +69,7 @@ const effects = {
 		const state = store.getState();
 
 		// Additional data needed for backwards compatibility.
-		// If we do not provide this data the post will be overriden with the default values.
+		// If we do not provide this data the post will be overridden with the default values.
```

Comment:
Given that we are changing this line maybe we can add also a comma:
If we do not provide this data, the post will be overridden with the default values.